lib/cAst.mli

```diff
@@ -0,0 +1,22 @@
+(************************************************************************)
+(*  v      *   The Coq Proof Assistant  /  The Coq Development Team     *)
+(* <O___,, *   INRIA - CNRS - LIX - LRI - PPS - Copyright 1999-2017     *)
+(*   \VV/  **************************************************************)
+(*    //   *      This file is distributed under the terms of the       *)
+(*         *       GNU Lesser General Public License Version 2.1        *)
+(************************************************************************)
+
+(** The ast type contains generic metadata for AST nodes *)
+type 'a ast = private {
```

Comment:
Do we gain anything by having `CAst.ast`? Wouldn't `CAst.t` work as well, i.e. being equally clear ?